prow/config.yaml

```
@@ -347,58 +347,36 @@ tide:
     - do-not-merge/work-in-progress
     - needs-rebase
   - orgs:
+    - kubernetes
     - kubernetes-client
     - kubernetes-csi
     - kubernetes-incubator
     - kubernetes-sigs
     repos:
     - client-go/unofficial-docs
-    - kubernetes/autoscaler
-    - kubernetes/client-go
-    - kubernetes/cloud-provider-aws
-    - kubernetes/cloud-provider-alibaba-cloud
-    - kubernetes/cloud-provider-azure
-    - kubernetes/cloud-provider-gcp
-    - kubernetes/cloud-provider-openstack
-    - kubernetes/cloud-provider-sample
-    - kubernetes/cloud-provider-vsphere
-    - kubernetes/cluster-registry
-    - kubernetes/community
-    - kubernetes/contrib
-    - kubernetes/dashboard
-    - kubernetes/dns
-    - kubernetes/enhancements
-    - kubernetes/examples
-    - kubernetes/federation
-    - kubernetes/frakti
-    - kubernetes/gengo
-    - kubernetes/git-sync
-    - kubernetes/ingress-gce
-    - kubernetes/ingress-nginx
-    - kubernetes/k8s.io
-    - kubernetes/klog
-    - kubernetes/kompose
-    - kubernetes/kops
-    - kubernetes/kube-deploy
-    - kubernetes/kube-openapi
-    - kubernetes/kube-state-metrics
-    - kubernetes/kubeadm
-    - kubernetes/kubectl
-    - kubernetes/kubernetes-anywhere
-    - kubernetes/kubernetes-template-project
-    - kubernetes/minikube
-    - kubernetes/node-problem-detector
-    - kubernetes/org
-    - kubernetes/perf-tests
-    - kubernetes/publishing-bot
-    - kubernetes/release
-    - kubernetes/repo-infra
-    - kubernetes/security
-    - kubernetes/sig-release
-    - kubernetes/steering
-    - kubernetes/test-infra
-    - kubernetes/utils
-    - kubernetes/website
+    excludedRepos:
+    - kubernetes/api
```

should we bother commenting that these are all staging repos? eg:
```suggestion
    - kubernetes/api # staging repos do not support merging
```

---

Yeah that sounds smart. Is every repo in the exclusion list a staging repo besides `k/k`?

---

> Is every repo in the exclusion list a staging repo besides `k/k`?

Yes, all of these are staging repos: https://github.com/kubernetes/kubernetes/tree/master/staging

---

ugh, well...there is one exception to "staging repos do not support merging": client-go is a staging repo but it should have Tide enabled because it still has some authoritative files in the published repo.

---

Thanks Nikhita. Updated with a comment.